application/versions.json

```diff
@@ -2,98 +2,155 @@
     "installed": [
         {
             "name": "behat/behat",
-            "version": "v3.10.0",
-            "description": "Scenario-oriented BDD framework for PHP"
+            "homepage": "http://behat.org/",
+            "source": "https://github.com/Behat/Behat/tree/v3.11.0",
+            "version": "v3.11.0",
+            "description": "Scenario-oriented BDD framework for PHP",
+            "abandoned": false
```

Comment:
These JSON blocks are getting to have so many fields that this diff is getting hard to read again.

Not necessary in this PR (unless you just want to), but it may be worth it for us to do something to simplify the contents of our `versions.json` file to make it easier (again) to see what the actual updates were to the direct dependencies we name in our composer.json file. Just a thought.

Reply:
Agreed. I think it would be a simple change. As commented above, just remove  `-f json` from the command in the Makefile.